test/unit/modules/blocks/verify.js

```diff
@@ -210,7 +210,7 @@ function createBlock (blocksModule, blockLogic, secret, timestamp, transactions,
 	return newBlock;
 }
 
-describe('blocks/verify', function () {
+describe.skip('blocks/verify', function () {
```

Comment:
@4miners why is this test skipped?

Reply:
@diego-G I explained that during core meeting yesterday.